### PR TITLE
[SIDE-805] Add Broker and StdIO services

### PR DESCRIPTION
**Pull request by richseviora**
This PR adds the Broker and STDIO services for consumption by non-Go clients.

### CODE REVIEW

**Review by BillyMurrayOlive — APPROVED** [review]

File: proto/broker.proto [hunk 1]
```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package plugin;
+option go_package = "plugin";
```
Comment (corylanou) [inline proto/broker.proto:3]: I'm not sure, but I think it should be 
```suggestion
option go_package = ".;proto";
```

File: proto/broker.proto [hunk 2]
```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package plugin;
```
Comment (corylanou) [inline proto/broker.proto:2]:
```suggestion
package proto;
```

File: proto/stdio.proto [hunk 3]
```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+package plugin;
+option go_package = "plugin";
```
Comment (corylanou) [inline proto/stdio.proto:3]:
```suggestion
package proto;
option go_package = ".;proto";
```

**Review by corylanou — CHANGES_REQUESTED** [review]
I think the package name needs to be `proto` to match the other files in this package (for go at least).

**Comment by richseviora** [comment]
You were right! I tried compiling them as-is and I got these errors and warnings:
```
protoc --proto_path=proto proto/*.proto --go_out=build/gen
2020/10/08 11:48:14 WARNING: Deprecated use of 'go_package' option without a full import path in "broker.proto", please specify:
        option go_package = ".;plugin";
A future release of protoc-gen-go will require the import path be specified.
See https://developers.google.com/protocol-buffers/docs/reference/go-generated#package for more information.

2020/10/08 11:48:14 WARNING: Deprecated use of 'go_package' option without a full import path in "stdio.proto", please specify:
        option go_package = ".;plugin";
A future release of protoc-gen-go will require the import path be specified.
See https://developers.google.com/protocol-buffers/docs/reference/go-generated#package for more information.

protoc-gen-go: Go package "." has inconsistent names plugin (broker.proto) and proto (filesystem.proto)
--go_out: protoc-gen-go: Plugin failed with status code 1.
```
Applying the changes you suggested fixed them.

**Review by corylanou — APPROVED** [review]

**Pull request merged**